pages/community/precice-workshop-2024.md

```diff
@@ -259,6 +259,56 @@ The cost of lunch, as well as coffee and snacks is included in the registration
 - 15:00-15:30: ☕️ Coffee break
 - 15:30-17:30: Hands-on user support
 
+## Program details
+
+### User introductions
+
+The official welcome to the workshop on Wednesday 25 will be followed by a round of introductions, so that everyone knows who to talk to during the workshop.
+[Submit](https://ipvs.informatik.uni-stuttgart.de/cloud/s/JWJ4toJF2qCScYL) one PDF slide (16:9, with name `surname-firstname.pdf`) with your name and some key details/pictures about your research till September 23, as we need to prepare a single slideshow.
+
+### World Café
+
+A classic for a preCICE Workshop, the preCICE [World Café](https://en.wikipedia.org/wiki/World_caf%C3%A9_(conversation)) is your opportunity to shape the future of preCICE. Expect topics such as Features, Usability, Community, Funding, Teaching, and This Workshop.
+
+### User support sessions
+
+It's a workshop and not a classical conference, as we also get to do things together. Meet the experts to discuss your current or planned projects, as well as to debug any current issues.
+
+On March 31, 10:00-15:00, we will have a _post-workshop user-support session_ to catch up on any progress.
+
+### preCICE training course
+
+If you are new to preCICE, this is the perfect way to get started. February 13 is for you, offering only training (parts I-III). Even if you joined this course in previous years, don't miss the new part IV on February 16.
```

Comment:
```suggestion
If you are new to preCICE, this is the perfect way to get started. September 24 is for you, offering only training (parts I-III). Even if you joined this course in previous years, don't miss the new part IV on September 25.
```

Reply:
The new part this year is part V, and it is actually community-contributed:

```suggestion
If you are new to preCICE, this is the perfect way to get started. September 24 is for you, offering only training (parts I-IV). Even if you joined this course in previous years, don't miss the new, community-contributed part V on September 25.
```